phpcs.ruleset.xml

```diff
@@ -2,9 +2,9 @@
 <ruleset name="WordPress Coding Standards for Plugins">
 	<description>Generally-applicable sniffs for WordPress plugins</description>
 
-	<rule ref="WordPress-Extra" />
+	<rule ref="WordPress-Core" />
 
 	<exclude-pattern>*/tests/*</exclude-pattern>
 	<exclude-pattern>*/vendor/*</exclude-pattern>
-	<exclude-pattern>*/dev-lib/*</exclude-pattern>
```

Comment:
@westonruter 

Please ignore this.

I was changing dev-lib name to build in my custom lib. There is no need to change.

Sorry, I have accidentally sent pull request. 

Amazing lib you have provided. I am using it to deploy my plugin. :)